Source/GSStoryboardTransform.m

```diff
@@ -0,0 +1,828 @@
+                }
+              RELEASE(document);
+            }
+          END_FOR_IN(children);
+        }
+      END_FOR_IN(array);
+    }
+  else
+    {
+      [NSException raise: NSInternalInconsistencyException
+                  format: @"No document element found in storyboard file"];
+    }
+}
+
+- (BOOL) isProcessedDocument: (NSXMLDocument *)xmlIn
+{
+  NSArray *docArray = [xmlIn nodesForXPath: @"document" error: NULL];
+  if ([docArray count] > 0)
+    {
+      NSXMLElement *docElem = (NSXMLElement *)[docArray objectAtIndex: 0];
+      NSXMLNode *a = [docElem attributeForName: @"processed"];
+      NSString *value = [a stringValue];
+      if (value != nil)
+        {
+          return YES;
+        }
+      else
+        {
+          NSXMLNode *new_attr = [NSXMLNode attributeWithName: @"processed"
+                                                      stringValue: @"true"];
+          [docElem addAttribute: new_attr];
+        }
+    }
+  return NO;
+}
+
+- (NSXMLElement *) createStoryboardProxyElementWithSelector: (NSString *)selector
+                                                     target: (NSString *)dst
+                                            segueIdentifier: (NSString *)ident
+                                                     sender: (NSString *)src
+                                                       kind: (NSString *)kind
+{
+  NSXMLElement *sbproxy = [NSXMLElement elementWithName: @"storyboardSeguePerformAction"];
+
+  NSXMLNode *pselector
+    = [NSXMLNode attributeWithName: @"selector"
+                       stringValue: selector];
+  NSXMLNode *ptarget
+    = [NSXMLNode attributeWithName: @"target"
+                       stringValue: dst];
+  NSString *pident_value = [[NSUUID UUID] UUIDString];
+  NSXMLNode *pident
+    = [NSXMLNode attributeWithName: @"id"
+                       stringValue: pident_value];
+  NSXMLNode *psegueIdent
+    = [NSXMLNode attributeWithName: @"identifier"
+                       stringValue: ident];
+  NSXMLNode *psender
+    = [NSXMLNode attributeWithName: @"sender"
+                       stringValue: src];
+  NSXMLNode *pkind
+    = [NSXMLNode attributeWithName: @"kind"
+                       stringValue: kind];
+  
+  [sbproxy addAttribute: pselector];
+  [sbproxy addAttribute: ptarget];
+  [sbproxy addAttribute: pident];
+  [sbproxy addAttribute: psegueIdent];
+  [sbproxy addAttribute: psender];
+  [sbproxy addAttribute: pkind];
+
+  return sbproxy;
+}
+
+- (NSMapTable *) processConnections: (NSArray *)connectionsArray
+                        withObjects: (NSXMLElement *)objects
+                       controllerId: (NSString *)src
+{
+  NSMapTable *mapTable = [NSMapTable strongToWeakObjectsMapTable];
+  
+  FOR_IN (NSXMLElement*, connObj, connectionsArray)
+    {
+      NSXMLElement *connections = (NSXMLElement *)connObj;
+      NSArray *children = [connections children]; // there should be only one per set.
+      
+      FOR_IN (id, obj, children)
```

Comment:
Here again you should be using the type NSXMLElement*

Reply:
Done